lib/permit_ecto/resolver.ex

```diff
@@ -17,6 +17,39 @@ defmodule Permit.Ecto.Resolver do
   import Permit.Helpers, only: [resource_module_from_resource: 1]
 
   @impl Permit.ResolverBase
+  def resolve(
+        subject,
+        authorization_module,
+        resource_module,
+        action,
+        %{use_loader?: true} = meta,
+        _
+      ) do
+    params_for_loader = %{
+      action: action,
+      resource_module: resource_module,
+      subject: subject,
+      params: meta[:params]
+    }
+
+    with {_, true} <-
+           {:pre_auth, authorized?(subject, authorization_module, resource_module, action)},
+         resource <- meta[:loader].(params_for_loader),
```

Comment:
If in `permit_phoenix` the `loader/1` function is defined as such:
```elixir
def loader(resolution_context) do
  case unquote(opts)[:loader] do
    nil -> nil
    loader -> {:loader, unquote(__MODULE__).loader(resolution_context, unquote(opts))}
  end
end
```
you could check here whether loader was not given at all, or it returned nil